io_scene_nif/modules/nif_import/property/geometry/niproperty.py

```diff
@@ -139,15 +141,23 @@ def process_nivertexcolor_property(self, prop):
         # TODO [property][mesh] Use the vertex color modes
         NifLog.debug("NiVertexColorProperty property processed")
 
-    def _find_or_create_material(self):
-        b_mats = self.b_mesh.materials
-        if len(b_mats) == 0:
-            b_mat = bpy.data.materials.new("")
-            self.b_mesh.materials.append(b_mat)
-            NifLog.debug("Created placeholder material to store properties in {0}".format(b_mat))
-        else:
-            b_mat = self.b_mesh.materials[0]
-            NifLog.debug("Reusing existing material {0} to store additional properties in {0}".format(b_mat))
-        return b_mat
+    def _find_or_create_material(self, name=""):
```

Comment:
Think this can be removed as the material is created and passed to the processor in the calling method.

Reply:
Right, exactly.

Reply:
Done